### DISCUSSION TITLE
Validation Batch size 2x than of training batch size.

### DISCUSSION THREAD

**Discussion by DieHumbleX** [TorchIO-project/torchio#473] [Q&A] [answered]
I am going through this tutorial ( https://colab.research.google.com/github/fepegar/torchio-notebooks/blob/main/notebooks/TorchIO_tutorial.ipynb#scrollTo=zzVqpgRf5eFN )

Is there any specific reason why Validation Batch size 2x that of training batch size?
It would be great if some one could mention the logic behind it.
Thanks in advance!

**Comment by fepegar** [comment, marked as answer]
Yes, there is a reason. In validation, you don't need to store the activations for backpropagation. Therefore, more memory is available and we can use a larger batch size. I have added a little comment about that.
